base/dcp_feed_type.go

```diff
@@ -12,14 +16,86 @@ import (
 // Cannot use the serverContext to retrieve this information, as the cbgt manager for a database is initialized
 // before the database is added to the server context's set of databases
 var cbgtCredentials map[string]cbgtCreds
+
+// cbgtBucketToDBName maps bucket names to DB names for DBs currently registered with CBGT. Necessary because in some
+// contexts we only have access to the bucket name.
+var cbgtBucketToDBName map[string]string
```

Comment:
What context do we only have a bucket name in?

This may not work if we have >1 database on a bucket and they're using different credentials/scopes.

The case I'm thinking of in particular is this:
- `db1` `bucket1` `scope1` `[col1,...]`
- `db2` `bucket1` `scope2` `[col1,...]`

Which could be a supported configuration - there's still only a single scope per db.

Reply:
`cbgt.GetPoolsDefaultForBucket` has signature `func(server, bucket string, scopes bool) ([]byte, error)`.

Reply:
As discussed, I'll start the conversation with the cbgt maintainers about changing this, but let's not block this PR on that.